grails-app/controllers/org/pih/warehouse/api/ApiController.groovy

```diff
@@ -197,7 +201,8 @@ class ApiController {
                 currencyCode         : currencyCode,
                 localizedHelpScoutKey: localizedHelpScoutKey,
                 isHelpScoutEnabled   : isHelpScoutEnabled,
-                localizationModeEnabled : localizationModeEnabled
+                localizationModeEnabled : localizationModeEnabled,
+                translationModeLocale : translationModeLocale
```

Comment:
This is my fault since I kept referring to this as "translation mode", but let's either use the existing name "localization mode" or convert everything to "translation mode".